examples/tutorials/tutorial_2D_gaw2.py

```diff
@@ -0,0 +1,135 @@
+geometry.group_faces_by_tag("faceName")
+geometry.group_edges_by_tag("edgeName")
+
+
+with SI_unit_system:
+    cylinders = [
+        fl.Cylinder(
+            name=f"cylinder{i+1}",
+            axis=[0, 1, 0],
+            center=[0.7, 0.5, 0],
+            outer_radius=outer_radius,
+            height=1.0,
+        )
+        for i, outer_radius in enumerate([1.1, 2.2, 3.3, 4.5])
+    ]
+    cylinder5 = fl.Cylinder(
+        name="cylinder5", axis=[-1, 0, 0], center=[6.5, 0.5, 0], outer_radius=6.5, height=10
+    )
+    farfield = fl.AutomatedFarfield(name="farfield", method="quasi-3d")
+    params = fl.SimulationParams(
+        meshing=fl.MeshingParams(
+            defaults=fl.MeshingDefaults(
+                surface_edge_growth_rate=1.17,
+                surface_max_edge_length=1.1,
+                curvature_resolution_angle=12 * u.deg,
+                boundary_layer_growth_rate=1.12,
+                boundary_layer_first_layer_thickness=2.8089171e-06,
+            ),
+            refinement_factor=1.35,
+            gap_treatment_strength=0.5,
+            volume_zones=[farfield],
+            refinements=[
+                fl.UniformRefinement(name="refinement1", spacing=0.1, entities=[cylinders[0]]),
+                fl.UniformRefinement(name="refinement2", spacing=0.15, entities=[cylinders[1]]),
+                fl.UniformRefinement(name="refinement3", spacing=0.225, entities=[cylinders[2]]),
+                fl.UniformRefinement(name="refinement4", spacing=0.275, entities=[cylinders[3]]),
+                fl.UniformRefinement(name="refinement5", spacing=0.325, entities=[cylinder5]),
+                fl.SurfaceRefinement(name="wing", max_edge_length=0.74, faces=[geometry["wing"]]),
+                fl.SurfaceRefinement(
+                    name="flap",
+                    max_edge_length=0.55,
+                    faces=[geometry["flap"]],
+                ),
+                fl.SurfaceRefinement(
+                    name="trailing",
+                    max_edge_length=0.36,
+                    faces=[
+                        geometry["wingTrailing"],
+                        geometry["flapTrailing"],
+                    ],
+                ),
+                fl.SurfaceEdgeRefinement(
+                    name="edges",
+                    method=fl.HeightBasedRefinement(value=0.0007),
+                    edges=[
+                        geometry["wingtrailingEdge"],
+                        geometry["wingleadingEdge"],
+                        geometry["flaptrailingEdge"],
+                        geometry["flapleadingEdge"],
+                    ],
+                ),
+                fl.SurfaceEdgeRefinement(
+                    name="symmetry", method=fl.ProjectAnisoSpacing(), edges=[geometry["symmetry"]]
+                ),
+            ],
+        ),
+        reference_geometry=fl.ReferenceGeometry(
+            moment_center=[0.25, 0.005, 0], moment_length=[1, 1, 1], area=0.01
```

Comment:
Same comment as in the tutorial_2D_30p30n.py file.

Reply:
Yes